src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -59,6 +59,8 @@ public class SnowflakeSinkConnector extends SinkConnector {
   // Using setupComplete to synchronize
   private boolean setupComplete;
 
+  private int VALIDATION_NETWORK_TIMEOUT = 45000;
```

Comment:
nit: `static final`

Reply:
Can you use TimeUnit.SECONDS.toMillis(45) to indicate this is seconds? 
Also `VALIDATION_NETWORK_TIMEOUT_SECONDS`

Reply:
Since the API takes in MS. I converted all to MS instead